### PR TITLE
Release 2.6.1

### PR DESCRIPTION
**Pull request by Anduin2017**

### CODE REVIEW

**Review by xxyzz — APPROVED** [review]

File: src/app/Views/about.html [hunk 1]
```diff
@@ -29,12 +29,12 @@ <h4>Kahla {{checkService.version}}</h4>
         <div class="text" i18n="@@CheckingForUpdates">Checking for updates...</div>
     </div>
 </ng-template>
-<div class="menu-button">
+<a class="menu-button" href="https://github.com/AiursoftWeb/Kahla.App/commits/master" target="_blank" rel="noopener noreferrer">
```
Comment (l2dy) [inline src/app/Views/about.html]: It would be better to display commits of the tagged release.

Comment (Anduin2017) [reply src/app/Views/about.html]: So how to do that?

Comment (ganlvtech) [reply src/app/Views/about.html]: What about <https://github.com/AiursoftWeb/Kahla.App/tags>?
Or we can write another file `UPGRADING` like [php upgrading history](https://github.com/php/php-src/blob/master/UPGRADING)

Comment (Anduin2017) [reply src/app/Views/about.html]: Good idea!

**Pull request merged**